master/setup.py

```diff
@@ -166,16 +170,17 @@ def define_plugin_entries(groups):
         "buildbot.steps.package.deb",
         "buildbot.steps.package.rpm",
         "buildbot.steps.source",
-        "buildbot.test",
+        "buildbot.test.util",
```

Comment:
Because they are part of the framework. They can be used to write tests for other projects using buildbot (e.g buildbot_travis)